hotsos/core/ycheck/engine/properties/search.py

```diff
@@ -43,11 +43,18 @@ def search_period_hours(self):
     @cached_yproperty_attr
     def search_result_age_hours(self):
         """
-        Result muct have aoccured within search-result-age-hours from current
-        time (in the case of a sosreport would time sosreport was created).
+        Result must have occurred within search-result-age-hours from current
+        time (in the case of a sosreport would be time sosreport was created).
         """
         return int(self.content.get('search-result-age-hours', 0))
 
+    @cached_yproperty_attr
+    def min_hours_since_last_boot(self):
+        """
+        Result must have occurred within hours from last boot.
```

Comment:
You mean: "[..] *after* hours [..]", right?

Maybe reword it to: "Result must have occurred after this many hours since last boot."?

Reply:
yes sorry my docstring says the opposite of what i meant :/ ill fix that now